module/mailnews/newsread.c

```diff
@@ -25,6 +26,15 @@ static list_type new_articles = NULL;   /* Build up new articles arriving from s
 
 static RECT  dlg_rect;        // Screen position of dialog
 
+// Constants for news dialog columns
+#define COL_TITLE 0x0000
```

Comment:
use static const int rather than macros for constants

Reply:
This compiles fine, but my editor is telling me this about the case value:
> expression must have a constant valueC/C++(28)
Using an enum solves this and compiles. Do you have a preference to keep these `static const int` or would an enum work?